tap_mambu/helpers/client.py

```diff
@@ -118,13 +120,21 @@ def __init__(self,
                  subdomain,
                  apikey_audit,
                  page_size,
-                 user_agent=''):
+                 user_agent='',
+                 window_size=DEFAULT_DATE_WINDOW_SIZE):
         self.__username = username
         self.__password = password
         self.__subdomain = subdomain
         base_url = "https://{}.mambu.com/api".format(subdomain)
         self.base_url = base_url
         self.page_size = page_size
+        try:
+            self.window_size = int(float(window_size)) if window_size else DEFAULT_DATE_WINDOW_SIZE
```

Comment:
Can't we do integer typecasting directly instead of, converting to float first?

Reply:
`>>> int("10.0")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ValueError: invalid literal for int() with base 10: '10.0'`